WcaOnRails/app/models/user.rb

```diff
@@ -56,7 +56,7 @@ def self.eligible_voters
 
   def self.leader_senior_voters
     team_leaders = TeamMember.current.in_official_team.leader.map(&:user)
-    senior_delegates = User.where(delegate_status: "senior_delegate")
+    senior_delegates = UserGroup.delegate_region_groups.where(parent_group_id: nil).map(&:lead_user)
```

Comment:
Might be worth moving `UserGroup.delegate_region_groups.where(parent_group_id: nil)` into its own `UserGroup.senior_delegate_region_groups` method.
If we _ever_ change the way senior regions are structured (for example, if in one year we decide to introduce a "World" Delegate level), you only have to touch that one method.

Reply:
Done